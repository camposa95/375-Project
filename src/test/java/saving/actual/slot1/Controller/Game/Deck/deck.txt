DevCards: [MONOPOLY, VICTORY, KNIGHT, MONOPOLY, KNIGHT, VICTORY, KNIGHT, KNIGHT, VICTORY, ROAD, PLENTY, KNIGHT, PLENTY, VICTORY, KNIGHT, KNIGHT, KNIGHT, KNIGHT, ROAD, KNIGHT, VICTORY, KNIGHT, KNIGHT, KNIGHT, KNIGHT]
