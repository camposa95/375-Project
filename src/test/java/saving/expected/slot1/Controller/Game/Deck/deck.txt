DevCards: [KNIGHT, VICTORY, VICTORY, PLENTY, MONOPOLY, KNIGHT, PLENTY, KNIGHT, VICTORY, KNIGHT, KNIGHT, VICTORY, KNIGHT, KNIGHT, KNIGHT, VICTORY, MONOPOLY, KNIGHT, KNIGHT, KNIGHT, ROAD, KNIGHT, ROAD, KNIGHT, KNIGHT]
